lib/sablon/html/ast.rb

```diff
@@ -8,6 +8,26 @@ def accept(visitor)
       def self.node_name
         @node_name ||= name.split('::').last
       end
+
+      private
+
+      # processes attributes defined on the node into wordML property syntax
+      def process_properties
```

Comment:
Can we push this logic into a `ParagraphProperties` class? I'm thinking something along the lines of:

```
ParagraphProperties.new(
  'numPr' => [
    { 'ilvl' => 1 }, { 'numId' => 24 }
  ]
)
```

When constructing a Paragraph node one would instantiate it directly with a `ParagraphProperties` instance:

```ruby
properties = ParagraphProperties.new({ ... })
Paragraph.new(properties, runs)
```

The `ParagraphProperties` class would have some convenience methods to interact with the underlying data:

```ruby
props = ParagraphProperties.new({ ... })
props.style # => 'Normal'
props.to_docx # => <w:.....
```